BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.h

```diff
@@ -5,20 +5,29 @@
 
 #import "BOXRequestWithSharedLinkHeader.h"
 
+typedef NS_OPTIONS(NSUInteger, BOXRepresentationRequestOptions) {
+    BOXRepresentationRequestNoOptions                          = 0,       // Default option if no options set or permission to file not granted
```

Comment:
Maybe we should just leave this out? `0` always behaves oddly in option sets, since you can't ask `x & 0 == 0` or anything like that.

Reply:
done